frontend/src/LandingPage.tsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import './LandingPage.css';
+import './UiElements.css';
+
+
+export class LandingPage extends React.Component<any, any>{
+
+    public render() {
+        return <div className="landing-site">
+            <div className="landing-site-inner">
+                <h1 className="landing-h1">Welcome to Supersocial</h1>
+                <h2 className="landing-h2">Social Media Marketing on a new level.</h2>
+                <div className="landing-box landing-flex">
+                    <img alt="Logo" className="landing-header-elem" src="logo512.png" />
+                    <div className="landing-header-elem landing-btn-wrapper">
+                        <div className="landing-header-text">Welcome to a new world of data-driven and AI-based social media marketing.</div>
+                        <div className="landing-flex-center">
+                            <button onClick={e => window.open("/register", "_self")} className="btn btn-primary landing-btn-header">Explore now &nbsp;&#9654;</button>
```

Comment:
`window.location.href = '…'` is a little more performant than `window.open('…', '_self')` because no a function invocation is involved in the former.

Reply:
Done